src/components/GalleryPage/Winners.js

```diff
@@ -134,6 +134,15 @@ function Winners() {
 			<Grid container spacing={8} justify='center'>
 				{winnerCards}
 			</Grid>
+			<Grid container justify="center">
+				<Button variant='contained' disableElevation color="secondary" component='a'
+					href="https://hoth8.devpost.com/project-gallery" target='_blank'
+					rel='noreferrer noopener'
+					style={{ textTransform: 'none', padding: '4px 1.5em',
```

Comment:
If you **are** going to use these styles, then I would recommend moving them somewhere (look at material UI's `makeStyles` API) to avoid duplicating code.